lib/linguist/languages.yml

```diff
@@ -4809,6 +4809,14 @@ Slim:
   codemirror_mode: slim
   codemirror_mime_type: text/x-slim
   language_id: 350
+SmPL:
+  type: programming
+  extensions:
+  - ".cocci"
+  ace_mode: text
+  tm_scope: none
+  color: "#c94949"
+  language_id: 164123055
```

Comment:
It might be worth including a few aliases:

<pre><code>aliases:
- coccinelle
<s>- ladybug</s>
- semantic patch language</code></pre>

Reply:
I've added *coccinelle*. I don't think anyone uses *semantic patch language*.